moskito-frontend/src/services/getInitialCampsiteFilter.js

```diff
@@ -6,6 +6,9 @@ export default function getInitialCampsiteFilter() {
             name: 'Postleitzahl/Ort',
             dbName: 'postalCode',
             postalCode: '',
+            distance: 100,
+            latitude: -1,
```

Comment:
Interesting that you start in the south atlantic ocean :-)

Reply:
oh good point, didn't know, that geocodes can be negative as well